com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -591,13 +477,18 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
                 publicKeyInputStream = new FileInputStream(manifestBuilder.getPublicKeyFilepath());
                 task.getOutputs().get(4).setContent(publicKeyInputStream);
 
+                // game.graph.json
+                fullGraph.setExcludedResources(new HashSet<>(excludedResources));
+                fullGraph.setHexDigests(archiveBuilder.getCachedHexDigests());
+                String fullGraphJSON = fullGraph.getRootNode().toJSON();
+                task.getOutputs().get(5).setContent(fullGraphJSON.getBytes());
```

Comment:
What is the path for this file?

Reply:
It ends up in the build folder, next to the archive etc. Possibly a bad choice...

Reply:
maybe it's fine. it's not like we have a lot of options:
- if it's controlled by `liveupdate` flag - then it's possible to put it near archive
- or your option with a build folder

- but if we add a separate option - we can require a path. But I think it's not necessary to have a separate option tbh